prowler/lib/outputs/jira/jira.py

```diff
@@ -98,6 +100,10 @@ class Jira:
     _client_id: str = None
     _client_secret: str = None
     _access_token: str = None
+    _client_mail: str = None
+    _api_token: str = None
+    _site_name: str = None
```

Comment:
what about `domain`? Please review how Atlassian names this.

Reply:
done, domain is the correct way to name it